support-frontend/test/selenium/contributions/OneOffContributionsSpec.scala

```diff
@@ -13,7 +14,14 @@ class OneOffContributionsSpec extends FeatureSpec with GivenWhenThen with Before
 
   override implicit val patienceConfig = PatienceConfig(Span(1, Minute), Span(5, Seconds))
 
-  before { driverConfig.reset() }
+  before {
+    driverConfig.reset()
+    // TODO - change tests to support new Stripe Elements UI
+    webDriver.asInstanceOf[JavascriptExecutor].executeScript(
+      """window.localStorage.setItem('gu.support.abTests',""" +
```

Comment:
Is this the only way to set local storage? Is there nothing equivalent to `driverConfig.addCookie()`?

Reply:
there is a way to access local storage, but I cannot get it to work properly. Not sure it's worth it
https://medium.com/@jonashavers/accessing-sessionstorage-and-localstorage-with-selenium-remotewebdriver-f3935d8d7d9b